usaspending_api/recipient/models.py

```diff
@@ -100,8 +100,8 @@ class Meta:
 class RecipientLookup(models.Model):
     recipient_hash = models.UUIDField(unique=True, null=True)
     legal_business_name = models.TextField(null=True, db_index=True)
-    duns = models.TextField(null=True)
-    parent_duns = models.TextField(null=True)
+    duns = models.TextField(unique=True, null=True, db_index=True)
```

Comment:
This index can only be a partial unique index, there are thousands of NULLs. See the Meta class for the existing indexes for both of these fields